project/dexter-monitor/public/ctrl/currentProjectCtrl.js

```diff
@@ -115,8 +138,12 @@ monitorApp.controller("CurrentProjectCtrl", function($scope, $http, $log, $inter
         if ($scope.refreshDataTimer) {
             $interval.cancel($scope.refreshDataTimer);
         }
-        if ($scope.dotAdder) {
-            $interval.cancel($scope.dotAdder);
+        if ($scope.timeDecreaser) {
+            $interval.cancel($scope.timeDecreaser);
         }
     });
+
+    $scope.reloadPage = function() {
+        $window.location.reload();
```

Comment:
Please change "$window.location.reload(true);"
you can check more information as below.
▶http://mwultong.blogspot.com/2006/08/html-windowlocationreloadtrue.html